### ISSUE TITLE
Currently does not support Linux Mint 21.3

### ISSUE THREAD

**Issue by th3oth3rjak3** [tmds/linux-dev-certs#15]
Tool fails for linux mint 21.3.

Repro:
```
dotnet linux-dev-certs install
Can not determine location to install CA certificate on Linux Mint 21.3
```
![Screenshot from 2024-05-10 19-57-04](https://github.com/tmds/linux-dev-certs/assets/77807678/15459f6b-f25b-4bcb-91e8-f2afc9ed8fa1)

**Comment by th3oth3rjak3** [comment]
It looks like this was fixed just a few days ago in the prerelease version. Sorry to make a duplicate issue. I'll mark this as closed.

**Comment by tmds** [comment]
The fix is part of the 0.1.1 version which was just uploaded to nuget.org.
